modules/core/src/viewports/viewport.js

```diff
@@ -246,6 +246,20 @@ export default class Viewport {
     );
   }
 
+  getFrustumPlanes(viewId) {
+    const {near, far, fovyRadians, aspect} = this.projectionProps;
+
+    return getFrustumPlanes({
```

Comment:
Should we save the frustum planes on the viewport the first time we ask for them? So that if the viewports haven't changed but something else triggers a redraw, the app gets the same array back?